session-signatures/getLitActionSessionSigs/src/index.ts

```diff
@@ -0,0 +1,103 @@
+import { getEnv } from "./utils";
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import {
+  AuthMethodScope,
+  LitNetwork,
+  ProviderType,
+  LIT_RPC,
+} from "@lit-protocol/constants";
+import {
+  EthWalletProvider,
+  LitAuthClient,
+} from "@lit-protocol/lit-auth-client";
+import {
+  LitAbility,
+  LitActionResource,
+  LitPKPResource,
+} from "@lit-protocol/auth-helpers";
+import * as ethers from "ethers";
+import { LocalStorage } from "node-localstorage";
+
+const ETHEREUM_PRIVATE_KEY = getEnv("ETHEREUM_PRIVATE_KEY");
+const LIT_RELAYER_API_KEY = getEnv("LIT_RELAYER_API_KEY");
+
+export const getSessionSigsLitAction = async () => {
+  let litNodeClient: LitNodeClient;
+
+  try {
+    const ethersSigner = new ethers.Wallet(
+      ETHEREUM_PRIVATE_KEY,
+      new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+    );
+
+    console.log("🔄 Connecting LitNodeClient to Lit network...");
+    litNodeClient = new LitNodeClient({
+      litNetwork: LitNetwork.DatilDev,
+      debug: false,
+      storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+      },
+    });
+    await litNodeClient.connect();
+    console.log("✅ Connected LitNodeClient to Lit network");
+
+    console.log(
+      "🔄 Initializing LitAuthClient for authentication through Lit login..."
+    );
+    const litAuthClient = new LitAuthClient({
+      debug: false,
+      litRelayConfig: {
+        relayApiKey: LIT_RELAYER_API_KEY,
+      },
+      rpcUrl: LIT_RPC.CHRONICLE_YELLOWSTONE,
+      litNodeClient,
+    });
+    console.log("✅ Successfully initialized LitAuthClient instance");
+
+    console.log(
+      "🔄 Authenticating EthWalletProvider, generating an Auth Method..."
+    );
+    const authMethod = await EthWalletProvider.authenticate({
+      signer: ethersSigner,
+      litNodeClient,
+    });
+    console.log(
+      "✅ Successfully authenticated EthWallerProvider, generated Auth Method"
+    );
+
+    console.log("🔄 Minting a PKP using Auth Method...");
+    const pkp = await litAuthClient.mintPKPWithAuthMethods([authMethod], {});
```

Comment:
Are we sure this is required? I don't think we need to specify the Auth Sig Auth Method here because `ethersSigner` will own the PKP and can authorize usage of it with an Auth Sig by default

I would think the Lit Action would need to be added as an Auth Method, but maybe not here because we're proving the code directly instead of an IPFS CID?

Reply:
I think you could just mint a PKP like so:

```ts
    console.log("🔄 Connecting LitContracts client to network...");
    const litContracts = new LitContracts({
      signer: ethersSigner,
      network: LitNetwork.DatilDev,
      debug: false,
    });
    await litContracts.connect();
    console.log("✅ Connected LitContracts client to network");

    console.log("🔄 Minting new PKP...");
    const mintedPkp = (await litContracts.pkpNftContractUtils.write.mint()).pkp;
    console.log(
      `✅ Minted new PKP with public key: ${mintedPkp.publicKey} and ETH address: ${mintedPkp.ethAddress}`
    );
```

then use `litNodeClient.getLitActionSessionSigs`

Reply:
Changed this to use `LitContracts`